src/main/java/com/ibm/watson/modelmesh/Metrics.java

```diff
@@ -230,6 +229,33 @@ public PrometheusMetrics(Map<String, String> params) throws Exception {
                 }
             }
 
+            if (!infoMetricParams.isEmpty()){
+                if (infoMetricParams.size() > INFO_METRICS_MAX) {
+                    throw new Exception("Too many info metrics provided in env var " + MMESH_CUSTOM_ENV_VAR + ": \""
+                            + infoMetricParams+ "\". The max is " + INFO_METRICS_MAX);
+                }
+
+                // Remove unset (unresolved) labels
+                for(String key : infoMetricParams.keySet()) {
```

Comment:
It is better to use `entrySet()` here.. to iterate over key/value pairs rather than look up each value.